app/src/main/java/com/threecsedevs/medichecker/activities/InteractionResultActivity.kt

```diff
@@ -110,9 +117,14 @@ class InteractionResultActivity : AppCompatActivity() {
 
 
     }
-    fun changeResultText(result: String?) {
+    fun changeResultText(result: MutableList<String>) {
         loadingCircle.visibility = View.GONE
-        resultTest.text = result
+        var text = ""
+        for (i in result){
+            text += i
+            text += "\n"
```

Comment:
Can we have more rich response? 
For example, like a bulleted list
* This medicine interacts with that medicine
* This ingredient will dissolve this medicine

Reply:
I think dash is okay too.
```
- Lorem ipsum dolor sit amet, consectetur adipiscing elit
- sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Reply:
I think Bulleted list is nice idea. I'll add it to the next commit.